data/src/main/kotlin/com/nexters/bandalart/data/local/datasource/GuestLoginTokenDataSourceImpl.kt

```diff
@@ -6,19 +6,22 @@ import androidx.datastore.preferences.core.edit
 import androidx.datastore.preferences.core.emptyPreferences
 import androidx.datastore.preferences.core.stringPreferencesKey
 import com.nexters.bandalart.data.datasource.GuestLoginTokenDataSource
-import com.nexters.bandalart.data.local.datastore.DataStoreKeys
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.catch
 import kotlinx.coroutines.flow.map
 import java.io.IOException
 
 class GuestLoginTokenDataSourceImpl(private val dataStore: DataStore<Preferences>) : GuestLoginTokenDataSource {
 
-  private val key = stringPreferencesKey(DataStoreKeys.GUEST_LOGIN_TOKEN)
+  companion object KEY {
```

Comment:
요기다가도 private 붙혀줘야 해 
```suggestion
private companion object KEY 
```

Reply:
완료!